### ISSUE TITLE
Quarkus IT: StartCommandDistTest#testHttpEnabled

### ISSUE THREAD

**Issue by miquelsi** [keycloak/keycloak#26147]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

[Quarkus IT (windows-latest, sanity-check-zip)](https://github.com/keycloak/keycloak/actions/runs/7441613988/job/20244141929)

`Caused by: java.lang.IllegalStateException: fv-az731-105-62530: failed to find an available port in ports [49935, 49936, 49937, 49938]`

### Version

latest

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

**Comment by shawkins** [comment]
@miquelsi how often is this happening? The mostly likely issue is that there's some process cleanup that is not happening, but it's not clear what - I don't think we've seen this on GHAs, and there's nothing unusual in the log proceeding this. 

It also seems like we're giving a pretty narrow port range to infinispan / jgroups - @mabartos @mhajas @vmuzikar is that intentional?

**Comment by miquelsi** [comment]
@shawkins so far this has happened only once. I'm fine with closing it for now and reopen if it happens again.

**Comment by shawkins** [comment]
@miquelsi sounds good.

**Comment by shawkins** [comment]
This is the same as #34844 and the subsequent incarnation. We have avoided defaulting to local cache for all our integration tests, so this will be an iterative process.

**Comment by shawkins** [comment]
Was already covered by #37089
